tests/transactions/e2e/features/send-money-to-receiver.feature

```diff
@@ -37,6 +38,7 @@ Feature: Send Money to Receiver
       | message                   |
       | api.errors.invalidAddress |
 
+  @api-wip-byron
```

Comment:
@tomothespian this test should be working as the "Send" screen is fully functional on Byron builds as well as on ITN builds.